tests/e2e/heatmap/heatmap.e2e.js

```diff
@@ -0,0 +1,17 @@
+import { Selector } from "testcafe";
+
+const path = require("path");
+
+import { testcafe_page_visual_regression } from "@curi-bio/frontend-test-utils";
+
+fixture`heatmap/basic`.page // declare the fixture
+`http://localhost:8080/heatmap/basic`; // specify the start page
+
+test("testing the HeatMap Layout NO option selected", async (t) => {
+  const screenshot_path_base = path.join("heatmap", "heatmap-layout-basic");
+  const screenshot_path = path.join(
+    screenshot_path_base,
+    "basic-heatmap-layout-no-selection"
+  );
+  await testcafe_page_visual_regression(t, screenshot_path);
```

Comment:
Sure, I will modify to match the Mockflow UI

Reply:
@jperson noticed a few other minor differences.  He's figuring out how to login to mockflow and be able to post those comments soon. But you can wait until he posts to make changes to this branch. Continue on the Y-Axis options in the meantime

Reply:
Testcafe Screenshot:
<img width="175" alt="Screen Shot 2021-03-22 at 9 54 59 AM" src="https://user-images.githubusercontent.com/236104/112027796-c82d5700-8af4-11eb-9def-da5c03a3649a.png">

Background color of text below Twitch Force box doesn't match the background color of the rest of the widget.

Reply:
We cannot fix the 'Twitch Force Box',  background color, there is a UI/UX design error, the at the moment we utilize a common widget 'InputDropDown' under which the definition of the background is rgb(0,0,0) so its re-used as a common module in any page, the UI/UX design has to maintain some form of consistent behaviour in all pages. This is not maintained so the difference is observed.

Reply:
Can you have the widget itself not have any specific background defined (for the region outside the input field...i.e. where the "An ID is required" text is), and let the parent page define the background CSS that sits underneath the widget?

Reply:
Can't understand what do you mean by specific background, there is already a background value coming from the basic DOM element called "input" which is set inside the widget, if we modify as per your suggestion all the input dropdown would also get effected. Let me know is that what we need to do ?

Reply:
In the InputDropDown.vue the code is as follows:-
```
<b-form-input
          :id="'input-dropdown-widget-' + options_id"
          v-model="input_dropdown_value_key"
          :list="'option_list' + options_id"
          :placeholder="placeholder"
          :disabled="disabled"
          class="w-100 h-100 edit-id"
          style="
            border-radius: 0;
            background-color: #1c1c1c;
            border: 0px;
            color: #ffffff;
          "
        ></b-form-input>
```
So  this is the value set into the Bootstrap via InputDropDownWidget.vue

Reply:
Following changes can be observed in the updated HeatMap Initial layout:-
a) The "Auto Scale" is aligned with the Checkbox.
b) The text "Maximum" and "Minimum" and the "Input box" have space between in the changed Heatmap layout UI.

Reply:
We're not referring to the #1c1c1c color inside the b-form-input.  It's the #000000 that's the background behind the "An ID Is Required" error message text.  The idea is to remove any specific background coloring behind that text in the widget component itself, so that the background color can be defined be the CSS in the parent component using that widget (i.e. currently in the Heat Map, the dropdown widget sites on top of a large grey rectangle as the background)

Reply:
Any way this is not the right approach of correction, the UI/UX design, is done with an idea of not even considering the widget approach, of defining the pages,  have been informing since **more than one year**, when the MockFlow doesn't specify anything and is done like a PPT screen, we start finding fault on the implementation which utilizes the optimized approach of widget, on faulty defined UI/UX. Looks like all the defects of the UI/UX design of improper definition, needs correction from the developer alone.

Reply:
Now I have fixed the PR UI/UX Design, please review and its not having any residue background color or border around.